tests/neg/i24250.scala

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Is it possible to break the habit of inserting an initial blank line in test sources?